jaxrs-tck/src/main/java/jakarta/ws/rs/tck/contextprovider/JsonbContextProviderIT.java

```diff
@@ -107,67 +127,73 @@ public final void shouldUseApplicationProvidedJsonbInstance() throws Interrupted
             assertThat(responsePojo.getSeenBy(), is(expectedWaypoints));
+        } catch (URISyntaxException ex) {
+            fail("The URL should have been formatted into a URI without an error.", ex);
         }
-
-        instance.stop().toCompletableFuture().get();
     }
 
     public static final class CustomJsonbProvider implements ContextResolver<Jsonb> {
         private final RuntimeType runtimeType;
-
+    
         private CustomJsonbProvider(final RuntimeType runtimeType) {
             this.runtimeType = runtimeType;
         }
-
+    
         public final Jsonb getContext(final Class<?> type) {
             if (!POJO.class.isAssignableFrom(type))
                 return null;
-
-            return JsonbBuilder.create(new JsonbConfig().withSerializers(new CustomSerializer()).withDeserializers(new CustomDeserializer()));
+    
+            return JsonbBuilder.create(new JsonbConfig().withSerializers(new CustomSerializer())
+                    .withDeserializers(new CustomDeserializer()));
         }
-
+    
         private final class CustomSerializer implements JsonbSerializer<POJO> {
             @Override
-            public final void serialize(final POJO pojo, final JsonGenerator generator, final SerializationContext ctx) {
+            public final void serialize(final POJO pojo, final JsonGenerator generator,
+                    final SerializationContext ctx) {
                 generator.writeStartObject();
-                generator.write("seenBy", String.format("%s,CustomSerializer(%s)", pojo.getSeenBy(), CustomJsonbProvider.this.runtimeType));
+                generator.write("seenBy", String.format("%s,CustomSerializer(%s)", pojo.getSeenBy(),
+                        CustomJsonbProvider.this.runtimeType));
                 generator.writeEnd();
             }
         }
-
+    
         private final class CustomDeserializer implements JsonbDeserializer<POJO> {
             @Override
-            public final POJO deserialize(final JsonParser parser, final DeserializationContext ctx, final Type rtType) {
+            public final POJO deserialize(final JsonParser parser, final DeserializationContext ctx,
+                    final Type rtType) {
                 final POJO pojo = new POJO();
-                pojo.setSeenBy(String.format("%s,CustomDeserializer(%s)", parser.getObject().getString("seenBy"), CustomJsonbProvider.this.runtimeType));
+                pojo.setSeenBy(String.format("%s,CustomDeserializer(%s)", parser.getObject().getString("seenBy"),
+                        CustomJsonbProvider.this.runtimeType));
                 return pojo;
             }
         }
     }
 
     public static final class POJO {
         private String seenBy;
-
+    
         public final String getSeenBy() {
             return this.seenBy;
         }
-
+    
         public final void setSeenBy(final String seenBy) {
             this.seenBy = seenBy;
         }
     }
 
+    @ApplicationPath("")
```

Comment:
> I don't know the exact specific but there's some autodiscovery going on with the `ApplicationPath`. Because of that `EchoApplication` get's picked up as a servlet. It would also be possible to register it in the `web.xml`, then we wouldn't need the `ApplicationPath`.